eras/allegra/impl/src/Cardano/Ledger/Allegra/Rules/Utxo.hs

```diff
@@ -138,6 +138,14 @@ instance
 data AllegraUtxoEvent era
   = UpdateEvent (Event (EraRule "PPUP" era))
   | TotalDeposits (SafeHash (EraCrypto era) EraIndependentTxBody) Coin
+  | -- | The UTxOs consumed and created by a signal tx
+    TxUTxODiff
+      ( (UTxO era)
+      , -- \^ UTxOs consumed
+        (UTxO era)
+      )
+
+-- \^ UTxOs created
```

Comment:
The comment at the bottom seems not attached to the pair's member. Is that intended?

Reply:
Problem is that it was intended as a haddock, but it is impossible to document elements of a tuple, so the haddock was turned into comment by fourmolu and then floated out in this funny way, since fourmolu isn't that great when dealing with comments